clusterloader2/testing/load/config.yaml

```diff
@@ -366,6 +388,129 @@ steps:
       action: gather
   {{end}}
 
+# BEGIN scheduler throughput
```

Comment:
We don't have those kind of comments anywhere else - maybe remove them?
The "step" name should basically fully describe it (if it's not, we should adjust step names).